### PR TITLE
Update archive-telus-network-data.md

### PR DESCRIPTION
**Pull request by Durga-SC**
Removed 24 hrs time since its real time . @markjjo

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/archive-telus-network-data.md [hunk 1]
```diff
@@ -28,11 +28,9 @@ The following overview explains the process of using a connector to archive TELU
 
 1. Your organization works with TeleMessage and TELUS to set up a TELUS Network connector. For more information, see [TELUS Network Archiver](https://www.telemessage.com/office365-activation-for-telus-network-archiver/).
 
-2. Once every 24 hours, SMS messages from your organization’s TELUS Network are copied to the TeleMessage site.
+2. The TELUS Network connector that you create in the Microsoft 365 compliance center connects to the TeleMessage site every day and transfers the SMS messages from the previous 24 hours to a secure Azure Storage location in the Microsoft Cloud. The connector also converts the content of SMS messages to an email message format.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/archive-telus-network-data.md:31]:
```suggestion
2. The TELUS Network connector that you create in the Microsoft 365 compliance center connects to the TeleMessage site every day and transfers the SMS messages from the previous 24 hours to a secure Azure storage location in the Microsoft cloud. The connector also converts the content of SMS messages to an email message format.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@Durga-SC Please review and implement the suggested changes. Thanks.

**Comment by markjjo** [comment]
This PR has been replaced by newer PR

**Pull request closed without merging**